setup.py

```diff
@@ -55,6 +55,7 @@
         (BASE_DIR, ['data/nssm_original.exe'])
     )],
     install_requires=['indy-plenum==1.13.0.dev1034',
+                      'importlib-metadata==1.7.0',
```

Comment:
Do we need to pin this to version `1.7.0`?
In plenum we use `importlib_metadata>=2.0` 
https://github.com/hyperledger/indy-plenum/blob/406afdeca1630be688f803a3cba15115faa20e2b/setup.py#L111